src/main/java/com/doeiqts/pizzavoter/domain/Vote.java

```diff
@@ -0,0 +1,74 @@
+package com.doeiqts.pizzavoter.domain;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Vote implements Comparable<Vote> {
+    private Integer count;
+    private Set<String> usersWhoVoted = new HashSet<>();
+
+    public Vote(String userName) {
+        this.count = 1;
+        this.usersWhoVoted.add(userName);
+    }
+
+    public Vote addToVote(String userName) {
+        if(!this.usersWhoVoted.contains(userName)) {
```

Comment:
Should be able to make the conditional based off the return value of `this.usersWhoVoted.add()`